server/src/completion/ICompletionItemLocation.ts

```diff
@@ -12,4 +12,14 @@ export interface ICompletionItemLocation {
      * The completion information.
      */
     completion: CompletionItem;
+    /**
+     * The first position from which we can suggest this item.
+     * First character of a function, for example.
+     */
```

Comment:
I had to check `getAvailableCompletionItems` to understand those comments. :sob:

```suggestion
     * Useful to avoid giving completion suggestions pertaining to symbols
     * that are out of the current scope.
     */
```